src/commands/toolbox/token/create.ts

```
@@ -21,115 +21,265 @@ import createTokenPrompt from '../../../prompts/createTokenPrompt.js'
+            throw new Error(
+                `Missing required information:\n${flagDescriptions.join('\n')}\n\n` +
+                'Please provide all required information or use --wizard for interactive mode.'
+            );
+        }
 
+        // Using non-null assertions (!) here is safe because:
+        // 1. We've already validated that these required fields exist above
+        // 2. If any were missing, we would have thrown an error
+        // 3. TypeScript can't infer this guarantee, so we need to tell it explicitly
+        return {
+            name: validateTokenName(flags.name!),
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)